app/controllers/forgotten_password_controller.js

```diff
@@ -40,8 +40,8 @@ module.exports = {
   newPasswordGet: (req, res) => {
     const id = req.params.id
     const render = (user) => {
-      if (!user) return errorView(req, res)
-      res.render('forgotten_password/new_password', {id: id})
+      if (!user) return renderErrorView(req, res)
```

Comment:
Somewhere in our style guide is to always use `{}` might be nice to add them since its updating the line but not necessary for approval.